lib/src/live_chat.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:live_chat/src/live_chat_platform_interface.dart';
+
+class LiveChat {
+  static final LiveChat _singleton = LiveChat._internal();
+
+  factory LiveChat() {
+    return _singleton;
+  }
+
+  LiveChat._internal();
+
+  Future<void> openChatView({
+    required String licenseId,
+    required String username,
+    required String email,
+    String? groupId,
+    Map<String, String>? customParameters,
+  }) async {
```

Comment:
Since we always use the same license Id , I think we can pass it when we create an object from the live chat, for group id also can
```
 factory LiveChat({String name}) {
    _instance.licenseId = licenseId
    return _instance;
  }
```